### PR TITLE
adds serves/makes field

### PR DESCRIPTION
**Pull request by tsop14**
**MODEL**
- creates `detailedServes` class which includes a field with portionType
- create sealed trait portion type which is either 'serves' or 'makes'
- uses circe to create JSON for adding to curated_recipe table:
```
{"portion": "ServesType", "quantity": {"to": 2, "from": 2}}
```

**UI**
- if the user checks 'No serving information', the serves fields are disabed. This preserves their values on the UI but will not post with form
- if the user selects either radio button AND leaves quantites empty, they cannot submit the form
- if the user does not check 'no serving information' AND has not selected a radio button they can still submit the form

required

TODO
- [ ] clean prod database; 36 recipes have been added to DB with old `Serves` model.

### CODE REVIEW

**Comment by tsop14** [comment]
#46

File: common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala [hunk 1]
```diff
@@ -20,6 +20,42 @@ case class CuratedRecipe(
...
 )
 
+case class DetailedServes(
+  portion: PortionType,
+  quantity: Serves
+)
+
+object DetailedServes {
+  def fromServes(s: Option[Serves]): Option[DetailedServes] = {
+    s match {
```
Comment (LATaylor-guardian) [inline common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala]: `s` is already an Option so you can map on it like so:

```
serves.map(s=> DetailedServes(ServesType, Some(s)))
```

**Comment by tsop14** [comment]
screen shot 2016-10-12 at 16 37 14

File: common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala (hunk 1 above)
Comment (tsop14) [inline common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala]: Done.

**Comment by LATaylor-guardian** [comment]
👍

**Pull request merged**